index.js

```diff
@@ -804,7 +848,9 @@ async function fetchThemeManifest(contents_url, branch, stars) {
  */
 async function getThemeRepos(page = 1) {
```

Comment:
Is it possible to pass the search query into the function? And same for the "get extensions" one. Would make for more readable code.

Reply:
I'll look into it ASAP

Reply:
I'm not sure it is possible, more accurately it _is_ possible, but a new request needs to be called (calling the `newRequest()` function), so in order to use search queries as parameters, I would have to nest the variable through:
```js
newRequest() => loadAmount() => loadPage() => getThemeRepos()
```
Which I'm pretty certain won't do any good in refactoring.

Reply:
Ah, yeah, that's fair. Some of the legacy code is a little monolithic.